packages/server/lib/browsers/index.ts

```diff
@@ -15,23 +15,39 @@ const debug = Debug('cypress:server:browsers')
 const isBrowserFamily = check.oneOf(BROWSER_FAMILY)
 
 let instance: BrowserInstance | null = null
+let instanceId = 0
```

Comment:
```suggestion
let attempt = 0
```

Reply:
Addresses in https://github.com/cypress-io/cypress/pull/25898/commits/b6ec574ce3f595ef8daefbe21a1946770ea6046f